CHANGELOG.md

```diff
@@ -6,6 +6,39 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+<<<<<<< HEAD
```

Comment:
Please remove the text related to merge conflict in this file.

Reply:
updated